.travis.yml

```
@@ -36,9 +23,8 @@ before_install:
     - echo session.gc_probability = 0 >> $INI
     - echo apc.enable_cli = 1 >> $INI
     - phpenv config-rm xdebug.ini || echo "xdebug not available"
-    - if [[ $TRAVIS_PHP_VERSION = 5.* ]]; then echo yes | pecl install -f apcu-4.0.11; fi
-    - if [[ $TRAVIS_PHP_VERSION = 7.* ]]; then echo yes | pecl install -f apcu_bc-1.0.4; fi
-    - if [[ $TRAVIS_PHP_VERSION = 7.* ]]; then echo yes | pecl install -f apcu-5.1.17; fi
+    - yes | pecl install -f apcu_bc-1.0.4
+    - yes | pecl install -f apcu-5.1.17
```

```suggestion
    - yes | pecl install -f apcu-5.1.19
```

If we want this to work on nightly as well.

---

not needed for this PR

---

Well, the current Travis config won't install APCu at all on nightly. Your version would try to install an incompatible APCu. imho, we should either bump APCu or restore the `if`.

---

good call, bumped